docs/usage/model_config.md

```diff
@@ -584,6 +585,45 @@ print(type(model2.pet))
 #> <class '__main__.Pet'>
 ```
 
+
+## Coerce Numbers to Strings
+
+Pydantic no longer allows number types (`int`, `float`, `Decimal`) to be coerced as type `str` by default.
+
+Set [`coerce_numbers_to_str=True`](../api/config.md#pydantic.config.ConfigDict.coerce_numbers_to_str) to enable coercing of numbers to strings.
+
+```py
+from pydantic import BaseModel, ConfigDict, ValidationError
+
+
+class Model(BaseModel):
+    value: str
+
+
+try:
+    print(Model(value=42))
+except ValidationError as e:
+    print(e)
+    """
+    1 validation error for Model
+    value
+      Input should be a valid string [type=string_type, input_value=42, input_type=int]
+    """
+
+
+class Model(BaseModel):
+    model_config = ConfigDict(coerce_numbers_to_str=True)
+
+    value: str
+
+
+repr(Model(value=42).value)
+#> "42"
+repr(Model(value=42.13).value)
+#> "42.13"
```

Comment:
Adding an example for `Decimal` as well :)

```suggestion
#> "42.13"
repr(Model(value=Decimal('42.13')).value)
#> "42.13"
```

Reply:
It's another import as well. I didn't want to bloat examples with a lot of boilerplate.